docs/users/cli-usage.md

```diff
@@ -147,6 +150,8 @@ Successfully started example-typo3-site
 Your application can be reached at: http://example-typo3-site.ddev.local
 ```
 
+You will now probably be interested into creating / adjusting DB credentials, the information how to do this is here: https://ddev.readthedocs.io/en/latest/users/developer-tools/#using-development-tools-on-the-host-machine.
+
```

Comment:
Suggestion: For those wanting/needing to connect to the database within the database container directly, please see the [developer tools page](https://ddev.readthedocs.io/en/latest/users/developer-tools/#using-development-tools-on-the-host-machine).